### DISCUSSION TITLE
Dig Quad AE+ goes offline after receiving DDP data.

### DISCUSSION THREAD

**Discussion by rledonne** [forkineye/ESPixelStick#659] [General]
I have a dig quad ae+ (with an external antenna) with 4 of the ports being used.
It was working fine when it was just a dig quad, but I recently received a AE+ so I can use a SD card to play sequences as a remote.  I have tried multiple recent CI builds, hoping it would make a difference.  I also have tried 2 different AE+'s, both are doing the same thing.

When I start sending display testing DDP data from XLights, or from FPP, it goes offline and will not come back until it is rebooted.  Even after the DDP data source is stopped, the device never comes back online.  I can watch the dig quad drop from the list of connected clients in my router as soon as it starts receiving DDP data, and never come back.  The only way it will reconnect is to be power cycled.

There are currently only 3 devices on the network, my computer, the DigQuad and a Kulp K16.  The computer is hardwired, the kulp and dig quad are wifi.  Everything has a strong connection, they are all in the same room.

Any suggestions for troubleshooting?

**Comment by MartinMueller2003** [comment]
Can you retest using the latest code at https://espixelstickwebflasher.from-ct.com:5000/
This is working in the latest builds
